metricshub-doc/src/site/markdown/configuration/configure-monitoring.md

```diff
@@ -828,7 +828,9 @@ loggerLevel: ...
 
 #### Configure Connector Variables
 
-In **MetricsHub**, connector variables are essential for customizing the behavior of data collection. The connector variables are configured in the `metricshub.yaml` file under the `variables` section of your configured resource. These variables are specified under the name of the connector to which they belong and contain key-value pairs. The key of each variable corresponds to a variable already configured in the connector.
+In **MetricsHub**, connector variables are essential for customizing the behavior of data collection. These variables are configured in the `additionalConnectors` section of your `metricshub.yaml` file. When configuring variables for connectors with variables, the connector becomes an additional connector, which is by default forced, and uses the variables configured by the user.
```

Comment:
We need to review this documentation 😉

Reply:
Let me know when I can start the doc review @CherfaElyes  and @NassimBtk  :)

Reply:
Thanks @CherfaElyes! @iguitton you can start the review, I am fine with the source code.